ClinicWeb/src/main/java/pl/infoshare/clinicweb/file/FileService.java

```diff
@@ -1,145 +1,93 @@
 package pl.infoshare.clinicweb.file;
 
-import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.core.type.TypeReference;
-import com.fasterxml.jackson.core.util.DefaultPrettyPrinter;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.databind.ObjectWriter;
 import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
 import org.json.simple.JSONArray;
-import org.json.simple.JSONObject;
 import org.json.simple.parser.JSONParser;
 import org.json.simple.parser.ParseException;
 
-
-import java.io.*;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 import java.nio.file.Files;
-import java.util.LinkedHashSet;
-import java.util.Scanner;
-import java.util.Set;
+import java.nio.file.Path;
+import java.util.List;
 
-import static java.nio.file.Paths.get;
+public class FileService implements FileRepository {
 
-public class FileService {
-    public static final String DOCTOR_PATH = "Clinic/src/main/resources/doctor.json";
-    public static final String PATIENT_PATH = "Clinic/src/main/resources/patient.json";
+    public static final String DOCTOR_PATH = "ClinicWeb/src/main/resources/doctors.json";
```

Comment:
```suggestion
    private static final String DOCTOR_PATH = "ClinicWeb/src/main/resources/doctors.json";
```
tą statyczną metodę przenióśłbym do serwisu docktora i przekazał w parametrze metody path do szukania danych w pliku, analogicznie z patient